scripts/publish-private.js

```diff
@@ -15,7 +15,7 @@ function getTaggedVersion() {
  * @param {string} tag
  */
 function publish(dir, tag) {
-  execSync(`npm publish --tag ${tag} ${dir}`, { stdio: "inherit" });
+  execSync(`pnpm publish ${dir} --tag ${tag}`, { stdio: "inherit" });
```

Comment:
Since I haven't attempted a publish, I've rearranged args to match the pnpm docs just in case the order is important.